ztools/http/filetransport.go

```diff
@@ -52,11 +52,14 @@ func newPopulateResponseWriter() (*populateResponse, <-chan *Response) {
 		ch: make(chan *Response),
 		pw: pw,
 		res: &Response{
-			Proto:      "HTTP/1.0",
-			ProtoMajor: 1,
-			Header:     make(Header),
-			Close:      true,
-			Body:       pr,
+			Protocol: Protocol{
+				Name:  "HTTP/1.0",
```

Comment:
this particular codepath isn't really used - it's for serving files while running a Go http server. For our http client, we use HTTP/1.1